src/screens/LoginPage/LoginPage.tsx

```
@@ -1,1026 +1,180 @@
-                          </p>
-                        )
-                      ) : null}
+              {/* Tab Navigation */}
+              <div className={styles.tabContainer} role="tablist">
+                <button
+                  role="tab"
+                  aria-selected={tab === 'LOGIN'}
+                  className={`${styles.tab} ${tab === 'LOGIN' ? styles.activeTab : ''}`}
+                  onClick={() => setTab('LOGIN')}
+                  data-testid="login-tab"
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)